tests/components/workday/test_config_flow.py

```diff
@@ -346,7 +346,7 @@ async def test_form_incorrect_dates(hass: HomeAssistant) -> None:
             CONF_OFFSET: DEFAULT_OFFSET,
             CONF_WORKDAYS: DEFAULT_WORKDAYS,
             CONF_ADD_HOLIDAYS: ["2022-12-12"],
-            CONF_REMOVE_HOLIDAYS: ["Weihnachtstag"],
+            CONF_REMOVE_HOLIDAYS: ["Christmas"],
```

Comment:
This would be quite a breaking change if you can not express removal of holidays based on local language anymore?

Reply:
Not sure I get it but the configuration should still be in local language regardless of system locale.
For example my own system is in English but I'm still Swedish and would like to write "Julafton" and not "Christmas"
Please change it back

Reply:
>For example my own system is in English but I'm still Swedish and would like to write "Julafton" and not "Christmas"

Thank you for the use case example.

>Please change it back

I don't think it works like that. The current python-holidays l10n changes the default holiday names output for some countries. So the component code needs to be adapted. That's what I meant by 

>// I guess more changes may be required to deal with the localized holidays.

I'll look into what can be done in order to integrate the new changes smoothly based on the example you provided.

Reply:
Ideally would be to be able to use both English and localized names.

Reply:
Any further progress here?